rest/blip_api_crud_test.go

```diff
@@ -1949,8 +1951,9 @@ func TestRemovedMessageWithAlternateAccess(t *testing.T) {
 
 	rt := NewRestTesterDefaultCollection(t, nil) // CBG-2619: make collection aware
 	defer rt.Close()
+	collection := rt.GetSingleTestDatabaseCollection()
 
-	resp := rt.SendAdminRequest("PUT", "/db/_user/user", `{"admin_channels": ["A", "B"], "password": "test"}`)
+	resp := rt.SendAdminRequest("PUT", "/db/_user/user", `{`+AdminChannelGrant(collection, `"admin_channels": ["A", "B"]`)+`, "password": "test"}`)
```

Comment:
Modifying AdminChannelGrant to set password would be more robust than the payload concatenation here, I think.